documentation/projects/proposals/project_improvement/20230914-implementation_plan_project_automations.md

```diff
@@ -0,0 +1,262 @@
+# Implementation plan: Project automations
```

Comment:
Do you mind adding the date to the title of this and the project plan? Potentially with the project plan one day earlier so they can be ordered semantically too? 😅 Example: https://docs.openverse.org/projects/proposals/additional_search_views/index.html

Reply:
Fixed in 2f61873.